src/service/repo/anonymousMessageRepo.ts

```diff
@@ -119,13 +122,28 @@ async function setChannelAsRead(channelId: string): Promise<boolean> {
   }
 }
 
+async function deleteMessage(messageId: string) {
+  try {
+    const response = await anonymousApi.delete(baseUrl.deleteMessage(messageId));
+    if (response.status === 200) {
+      return Promise.resolve(response.data);
+    }
+
+    return Promise.reject(response.data?.status);
+  } catch (e) {
+    console.log(e);
+    return Promise.reject(e);
+  }
+}
+
 const AnonymousMessageRepo: AnonymousMessageRepoTypes = {
   checkIsTargetAllowingAnonDM,
   sendAnonymousMessage,
   getAllAnonymousChannels,
   getAllAnonymousPostNotifications,
   getSingleAnonymousPostNotifications,
-  setChannelAsRead
+  setChannelAsRead,
+  deleteMessage
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

In the `deleteMessage` function, you are logging the error directly to the console. This might not be the best approach for handling errors, especially in production code. Consider using a logging service that can provide more context about the error and help with debugging. Also, it would be better to throw an error instead of returning a rejected promise. This way, the caller of this function will have to handle the error, making it less likely that the error will be ignored.
```diff
-    console.log(e);
-    return Promise.reject(e);
+    // Use your preferred logging service here
+    logger.error(e);
+    throw e;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->